db/dbformat.cc

```diff
@@ -279,4 +279,16 @@ void IterKey::EnlargeBuffer(size_t key_size) {
   buf_ = new char[key_size];
   buf_size_ = key_size;
 }
+
+void IterKey::EnlargeSecondaryBufferIfNeeded(size_t key_size) {
+  // If size is smaller than buffer size, continue using current buffer,
+  // or the static allocated one, as default
```

Comment:
Very minor but seems to me the buffers are not actually statically allocated; maybe call them something like "fixed-size" or "inline"